lib/Backend/NativeCodeGenerator.cpp

```diff
@@ -2422,34 +2422,30 @@ NativeCodeGenerator::GatherCodeGenData(
                 Js::FunctionCodeGenRuntimeData *const inlineeRuntimeData = IsInlinee ? runtimeData->EnsureLdFldInlinee(recycler, inlineCacheIndex, inlineeFunctionBody) :
                     functionBody->EnsureLdFldInlineeCodeGenRuntimeData(recycler, inlineCacheIndex, inlineeFunctionBody);
 
-                    if (inlineeRuntimeData->GetFunctionBody() != inlineeFunctionBody)
-                    {
-                        //There are obscure cases where profileData has not yet seen the polymorphic LdFld but the inlineCache has the newer object from which getter is invoked.
-                        //In this case we don't want to inline that getter. Polymorphic bit will be set later correctly.
-                        //See WinBlue 54540
-                        continue;
-                    }
-
+                if (inlineeRuntimeData->GetFunctionBody() != inlineeFunctionBody)
+                {
+                    //There are obscure cases where profileData has not yet seen the polymorphic LdFld but the inlineCache has the newer object from which getter is invoked.
+                    //In this case we don't want to inline that getter. Polymorphic bit will be set later correctly.
+                    //See WinBlue 54540
+                    continue;
+                }
 
-                    if (!isJitTimeDataComputed)
```

Comment:
Because this bool will always be false for non-recursive inlining and I disabled recursive inlining for functions having accessor calls.